backend-config/account_association_test.go

```diff
@@ -238,7 +240,102 @@ func TestAccountAssociations(t *testing.T) {
 
 		c.processAccountAssociations()
 
-		require.Equal(t, "acc-1", c.Sources[0].Destinations[0].DeliveryAccount.Id)
-		require.Empty(t, c.Sources[0].Destinations[0].DeliveryAccount.AccountDefinition)
+		require.Equal(t, "acc-1", c.Sources[0].Destinations[0].DeliveryAccount.ID)
+		require.Nil(t, c.Sources[0].Destinations[0].DeliveryAccount.AccountDefinition)
+	})
+
+	t.Run("blank account definition name", func(t *testing.T) {
```

Comment:
Can you add more description in the test case like what is the expectation for this scenario

Reply:
Like we will expect an associated account with destination but it has nil accountDefinition